photon-ml/src/main/scala/com/linkedin/photon/ml/util/IOUtils.scala

```diff
@@ -21,17 +21,17 @@ import com.linkedin.photon.ml.avro.AvroIOUtils
 import com.linkedin.photon.ml.io.GLMSuite
 import com.linkedin.photon.ml.stat.BasicStatisticalSummary
 import com.linkedin.photon.ml.supervised.model.GeneralizedLinearModel
-
 import scala.collection.mutable
+
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.Path
 import org.apache.spark.SparkContext
 import org.joda.time.Days
-
 import java.lang.{Double => JDouble}
 import java.util.{Map => JMap}
 
 import scala.collection.JavaConversions._
+import scala.util.{Failure, Success, Try}
```

Comment:
Import sections should be:
```
java
javax

scala

external libs

com.linkedin.photon
```
This comes from the [Databricks Spark Scala style guide](https://github.com/databricks/scala-style-guide#imports)